### PR TITLE
Display tagline on mobile

### LINKED ISSUE #1994: Mobile view:  Some text is not displayed
The second line of the tagline  "Advance your career. Accelerate your master’s degree"  is missing in mobile view

 
mobile

### PR DESCRIPTION
**Pull request by singingwolfboy**
Fixes #1994.
mobile-1
mobile-2

### CODE REVIEW

File: static/scss/homepage.scss [hunk 1]
```diff
@@ -221,7 +221,9 @@ body.app-media {
       }
 
       @include breakpoint(phone) {
-        display: none;
+        font-size: 20px;
+        line-height: 1.15;
```
Comment (rhysyngsun) [inline static/scss/homepage.scss]: Should `line-height` have an `em` unit? I notice a style a few lines above lacks it too, but that probably needs to get corrected too.

Comment (singingwolfboy) [inline static/scss/homepage.scss]: @rhysyngsun [According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height), you _can_ provide a unit, [but it's better not to](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values).

Comment (rhysyngsun) [inline static/scss/homepage.scss]: Sounds good.

**Comment by rhysyngsun** [comment]
:+1:

**Comment by roberthouse54** [comment]
Can you add margin-bottom 10px to the h1 element for the mobile view. Other than that looks good.

**Comment by singingwolfboy** [comment]
@roberthouse54 Added. Is that better?

**Comment by roberthouse54** [comment]
Looks good 👍

**Pull request merged**